### PR TITLE
Improve default names for BETWEEN ? AND ? to have prefixes from_ and to_

### PR DESCRIPTION
**Pull request by Jille**
issue #2277

### CODE REVIEW

**Comment by Jille** [comment]
My change to the number was just random code cleanup, but I think that fixed support for BETWEEN sqlc.arg(min) AND sqlc.arg(max). Added a test case to keep it that way.

File: internal/endtoend/testdata/between_args/mysql/go/query.sql.go [hunk 1]
```diff
@@ -16,12 +16,41 @@ WHERE   price BETWEEN ? AND ?
...
+	return items, nil
+}
+
+const getBetweenPricesNamed = `-- name: GetBetweenPricesNamed :many
+SELECT  id, name, price
+FROM    products
+WHERE   price BETWEEN sqlc.arg(min_price) AND sqlc.arg(max_price)
+`
+
+func (q *Queries) GetBetweenPricesNamed(ctx context.Context) ([]Product, error) {
```
Comment (kyleconroy) [inline internal/endtoend/testdata/between_args/mysql/go/query.sql.go:52]: I think your change has inadvertently broken this code. Notice that this function now has zero arguments. I think you need to keep the code that picked the number for the argument.

**Review by kyleconroy — CHANGES_REQUESTED** [review]

File: internal/endtoend/testdata/between_args/mysql/go/query.sql.go (hunk 1 above)
Comment (Jille) [reply internal/endtoend/testdata/between_args/mysql/go/query.sql.go:52]: Actually, that never worked. #2277 was filed about that including https://play.sqlc.dev/p/7ecf294cb7aa0b3ce1025657e091cce22970d3b4f5c4eb2baa578e5083cd4219 which shows it doesn't work.

I thought I had fixed it, but maybe I looked wrong. I'll drop the second commit [the one adding this broken test case] from this PR for now and will have a look.

Comment (Jille) [reply internal/endtoend/testdata/between_args/mysql/go/query.sql.go]: It needed an extra fix. I made that in https://github.com/kyleconroy/sqlc/pull/2373.

**Review by kyleconroy — APPROVED** [review]

**Pull request merged**